arq/worker.py

```diff
@@ -133,6 +133,7 @@ class Worker:
     :param burst: whether to stop the worker once all jobs have been run
     :param on_startup: coroutine function to run at startup
     :param on_shutdown: coroutine function to run at shutdown
+    :param no_sig: does not register signal handlers, used when running inside other async framework
```

Comment:
```suggestion
    :param sigs: default True, whether register signal handlers, set to False when running inside other async framework
```
I don't like double negatives `not self._no_sig`, even when the new `False` mode is a change.